ti/instrumentation/instrumentation.go

```diff
@@ -166,6 +166,7 @@ func GetCmd(ctx context.Context, config *api.RunTestConfig, stepID, workspace st
 	// Ignore instrumentation when it's a manual run or user has unchecked RunOnlySelectedTests option
 	isManual := IsManualExecution(cfg)
 	ignoreInstr := isManual || !config.RunOnlySelectedTests
+	cfg.SetIgnoreInstr(ignoreInstr)
```

Comment:
So is this going to remove TI support for manual triggers?

Reply:
Nope, TI for manual triggers will stil work

Reply:
Yes we will not go for test selection if it a manual run (according to our old definition of manual run). Now since TI works for manual runs it will go